trade-finance-manager-ui/server/controllers/case/index.test.js

```diff
@@ -79,11 +87,44 @@ describe('controllers - case', () => {
           activeSubNavigation: 'deal',
           dealId: req.params._id,
           user: session.user,
+          showDealCancelButton: false,
           hasAmendmentInProgress: true,
           amendments: mockAmendments,
           amendmentsInProgress: mockAmendments,
         });
       });
+
+      it('should check whether deal cancellation is enabled', async () => {
+        await caseController.getCaseDeal(req, res);
+
+        expect(dealCancellationEnabled).toHaveBeenCalledTimes(1);
+      });
+
+      it('should render the template with the deal cancellation value when it is true', async () => {
+        jest.mocked(dealCancellationEnabled).mockReturnValueOnce(true);
+
+        await caseController.getCaseDeal(req, res);
+
+        expect(res.render).toHaveBeenCalledWith(
+          'case/deal/deal.njk',
+          expect.objectContaining({
+            showDealCancelButton: true,
+          }),
+        );
+      });
+
+      it('should render the template with the deal cancellation value when it is false', async () => {
```

Comment:
For both of these tests I think some describe blocks would be nice:

```js
describe('when deal cancellation is enabled'
  it('should render the template with showDealCancelButton=true'

describe('when deal cancellation is disabled'
  it('should render the template with showDealCancelButton=false'
```

Reply:
Good idea, have changed

Reply:
Resolving this as implemented👍